### PR TITLE
ANXOS-207 Move organization to anexia

### PR DESCRIPTION
**Pull request by pascalski**
This PR moves prepares the repo for a move to the anexia organization. Currently it is under anexia-it

### CODE REVIEW

**Comment by pascalski** [comment]
@nezhar Please review. This PR prepares the repo for the move to anexia

File: README.rst [hunk 1]
```diff
@@ -2,6 +2,11 @@
 Django Request Cache
 ====================
 
+.. image:: https://badge.fury.io/py/django-request-cache.svg
+    :target: https://badge.fury.io/py/django-request-cache
+.. image:: https://github.com/anexia/django-request-cache/actions/workflows/test.yml/badge.svg?branch=master
```
Comment (nezhar) [inline README.rst:7]: Can you please also add a badge for `codecov` and use `main` as the default branch instead of `master`?

Comment (pascalski) [reply README.rst]: Changed :+1:

**Comment by pascalski** [comment]
@nezhar I've added the codecov badge and changed your comment :+1:

**Review by nezhar — APPROVED** [review]

**Pull request merged**